app/jobs/reports/quarterly_account_stats.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+require 'csv'
+
+module Reports
+  class QuarterlyAccountStats < BaseReport
+    REPORT_NAME = 'quarterly-account-stats'
+
+    # put this in job_configurations.rb
+
+    def perform(report_date)
+      report = report_body(report_date - 90.days, report_date)
+      puts report
+      # call save_report
+    end
+
+    def report_body(start_date, end_date)
+      report = {}
+
+      report[:start_date] = start_date.to_s
+      report[:end_date] = end_date.to_s
+
+      report[:deleted_users_all_time] = deleted_user_count
+      report[:deleted_users_for_period] = deleted_user_count(start_date:, end_date:)
+
+      report[:users_all_time] = user_count
+      report[:users_for_period] = user_count(start_date:, end_date:)
```

Comment:
let's label this `total_registered_users` to [match our query definitions](https://handbook.login.gov/articles/queries.html#total-registered-including-deletes)

Reply:
I'm not sure I agree, unless we have a better strategy for the other names.

First, the link goes to "Total Registered, Including Deletes," possibly in error, but also showing that this is already confusing business.

The pattern used on the others is to have `THING_all_time` and `THING_for_period` to differentiate the two. (I'm open to renaming that if there's something clearer.) Calling the first one `total_registered_users_all_time` makes sense, but `total_registered_users_for_period` is potentially surprising if you're expecting "total registered users" to be the... total number on a given date, rather than those that signed up in a 90-day window.

I'm definitely open to clearer names, or really even being told that I'm wrong because I don't understand the historical background here, but it felt wrong when I went to implement this.